code/modules/antagonists/heretic/knowledge/ash_lore.dm

```diff
@@ -26,19 +26,36 @@
  *
  * Ashlord's Rite
  */
```

Comment:
These big ugly comments can be deleted now btw

Reply:
gonna do that after diff equasion classes